test/e2e/usersignup_test.go

```diff
@@ -45,7 +45,25 @@ func (s *userSignupIntegrationTest) TearDownTest() {
 	memberAwait2.Clean(s.T())
 }
 
-func (s *userSignupIntegrationTest) TestAutomaticApproval() {
+func (s *userSignupIntegrationTest) TestHomeSpaceStatusUpdate() {
```

Comment:
Same here. Instead of adding a brand new test I would rather update the existing test(s) to check the home space status there.

Reply:
Made it as a sub-test. Please check again.

Reply:
I was thinking about adding the home space status check here: https://github.com/codeready-toolchain/toolchain-e2e/blob/master/testsupport/user_assertions.go#L229
This function is used in many tests which create spaces. So there would be no need to add more tests if we added the check to that function.